.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 *.ipr
 *.iws
 /.idea
+myConfig.js
```

Comment:
Why did we add this to the .gitignore files? What does this file drive?

Reply:
My creds for waterbears-dev are included in this file